src/tests/pages/ActorPage.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { useAppSelector } from '../../store/store';
+import ActorPage from '../../pages/ActorPage/ActorPage';
+import renderWithProviders from '../../mockedData/test-utils';
+import dataActor from '../../mockedData/data-actor.json';
+import dataCards from '../../mockedData/data-movies.json';
+import dataImages from '../../mockedData/data-images.json';
+
+jest.mock('../../store/store', () => ({
+  __esModule: true,
+  ...jest.requireActual('../../store/store'),
+  useAppSelector: jest.fn(),
+}));
+
+describe('Actor page', () => {
+  beforeEach(() => {
+    (useAppSelector as jest.Mock).mockReturnValue({
+      lang: 'en',
+      genresArray: [{ id: 0, name: 'super action' }],
+      loading: false,
+      data: dataActor,
+      images: null,
+      films: dataCards,
+      error: '',
+    });
+  });
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+  it('should show loader component', () => {
+    (useAppSelector as jest.Mock).mockReturnValue({
```

Comment:
Also, if you need to change something that will look like (useAppSelector as jest.Mock).mockReturnValue(getUseAppSelectorMock({loading: true}))